azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/ParticipantInfoModel.kt

```diff
@@ -15,7 +15,7 @@ internal enum class ParticipantStatus {
 }
 
 internal data class ParticipantInfoModel(
-    val displayName: String,
+    var displayName: String,
```

Comment:
ParticipantInfoModel is in redux state right?

Everything should technically be val (Redux state should be immutable)

Reply:
Adam, info models are created in service layer and cached. When display name changed(can change while in call), info model is updated thus few are var.

Redux is also keeping list but we have copy in middleware I think. If required we can separate object for redux and service layer.